lib/server.js

```diff
@@ -36,6 +35,8 @@ function Server(opts){
   this.clientsCount = 0;
 
   opts = opts || {};
+
+  this.wsEngine = opts.wsEngine || process.env.EIO_WS_ENGINE || 'ws';
```

Comment:
For the record (i replied somewhere else already):
EIO_WS_ENGINE for engine.io
EIO_CLIENT_WS_ENGINE for engine.io-client